mitodl/micromasters
# Show program certificate on dashboard

#### What are the relevant tickets?
Fix #3521

#### What's this PR do?
If the learner earned a program certificate show it in the ProgressWidget.

#### How should this be manually tested?
Create a program certificate (follow instructions in this PR #3528). You should be able to see in your dashboard.

![screen shot 2017-09-21 at 4 21 41 pm](https://user-images.githubusercontent.com/7574259/30716741-00b4dbee-9ee9-11e7-895c-f66cd61384e9.png)
